access-control-app/src/cmr/access_control/services/acl_util.clj

```diff
@@ -113,9 +113,13 @@
           concept-ids (:concept-ids collection-identifier)
           provider-id (get-in acl [:catalog-item-identity :provider-id])
           colls-from-entry-titles (when (seq entry-titles)
-                                    (mdb1/find-concepts context {:provider-id provider-id :entry-title entry-titles} :collection))
+                                    (for [batch (mdb1/find-in-batches context :collection 100 {:provider-id provider-id :entry-title entry-titles})
```

Comment:
Also we discussed for testing, def the batch-size, change it to something small for the test, and have more than that # of collections so we know it hits the batching code.